src/toolbar/toolbar.html

```diff
@@ -1,8 +1,9 @@
 <div id="toolbar">
     <img id="logo" src="assets/svg/scottlogic_logo.svg" />
     <toolbar-icon name="minimise" icon-click="toolbarCtrl.minimiseClick()"></toolbar-icon>
-    <toolbar-icon name="maximise" ng-show="!toolbarCtrl.maximised" icon-click="toolbarCtrl.maximiseClick()"></toolbar-icon>
-    <toolbar-icon name="normal_size" ng-show="toolbarCtrl.maximised" icon-click="toolbarCtrl.normalSizeClick()"></toolbar-icon>
+    <toolbar-icon name="summarise" icon-click="toolbarCtrl.summariseClick()"></toolbar-icon>
+    <toolbar-icon name="maximise" ng-show="!toolbarCtrl.maximised && !toolbarCtrl.summarised" icon-click="toolbarCtrl.maximiseClick()"></toolbar-icon>
+    <toolbar-icon name="normal_size" ng-show="toolbarCtrl.maximised && !toolbarCtrl.summarised" icon-click="toolbarCtrl.normalSizeClick()"></toolbar-icon>
```

Comment:
<minor comment> This might be too much logic for HTML. Though I'm not overly concerned (it's just an `&&` after all..). If @WPFerg thinks so too might be worth making bespoke properties/functions.